shared/js/background/before-request.es6.js

```diff
@@ -323,7 +328,7 @@ function blockHandleResponse (thisTab, requestData) {
 
             console.info('blocked ' + utils.extractHostFromURL(thisTab.url) +
                         // @ts-ignore
-                        ' [' + tracker.tracker.owner.name + '] ' + requestData.url)
+                        ' [' + tracker.tracker.owner.name + '] ' + requestData.url + (requestData.tabId === -1 ? ' (serviceworker)' : ''))
```

Comment:
Nit: Assign `serviceWorkerInitiated` bool to a variable above, then you can pass it through to the postDevToolsMessage with the Object shorthand. Also, then you can reuse the variable here to avoid doing the `tabId === -1` check again.